docs/en/plugins/README.md

```diff
@@ -0,0 +1,413 @@
+Loomio’s recently released a plugin architecture, which will allow you quickly and easily add functionality to Loomio on your own instance. There’s some good docs in (plugin README), but I thought it might be helpful to step through how we make plugins here, step by step
+
+#### WHAT WE’LL BE MAKING:
+
+The thing we’ll be making this time is a tagging plugin, which will allow group coordinators to specify tags, which can then be used to classify discussions which are related, but distinct (Think something like ‘Board meeting minutes’, or ‘The Party Planning Committee’)
+
```

Comment:
The thing we’ll be making this time is a tagging plugin, which will allow group coordinators to specify tags, which can then be used to classify discussions which are related, but distinct (~~T~~**t**hink something like ‘Board meeting minutes’, or ‘The Party Planning Committee’)**.**